### ISSUE TITLE
will this work on 9.2?

### ISSUE THREAD

**Issue by timX24968B** [yellows8/ctr-httpwn#3]
I tried it on 9.2 on my O3ds and it flipped out and started corrupting stuff on the sd card.

**Comment by squee666** [comment]
https://i.gyazo.com/8b8dd977b5814799cdbb57e943ab7981.png

Did you do this?

**Comment by yellows8** [comment]
https://github.com/yellows8/ctr-httpwn#supported-sysmodule-versions

Not sure why you'd use this on v9.2 with totalcontrolhaxx though...

"started corrupting stuff on the sd card." Be more specific.

**Comment by timX24968B** [comment]
also, I used menuhax.  i tried browserhax once, it was a pain in the ass

**Comment by Nio9345** [comment]
What is this totalcontrolhax for 9.2 ? I'm asking because I also have an issue with this trick as well, just mine it says that the sysmodule v13318 isn't supported (pressumably my FW isn't supported by this trick).

**Comment by timX24968B** [comment]
I have not been able to find "totalcontrolhax", could you reference it somewhere?

**Comment by timX24968B** [comment]
When i said corrupting the SD card, i would run the app, and all it would do is open another folder in the hbmenu containing another folder (with a junk name) and rf launcer. Opening that junk name folder led to no other folders, just the rf launcher. After this, I turned off my 3ds and checked the sd card to make sure everything was fine. There was one single 3gb file with a random character for the name of it instead of the .3dsx and .xml files. I tried deleting it, did not let me. I tried putting the .3dsx and .xml files back on, and it said that one of the files was corruted. This is where it just made more of those weirdly named files, and when i took the sd card out and put it back in, windows said the drive needed to be repaired. I let it repair, and at the end, there was no folder for ctr-httpwn.

**Comment by yellows8** [comment]
"corrupting the SD card" That really has nothing to do with this app since it was never launched.

**Comment by yellows8** [comment]
"pressumably my FW isn't supported" It says so in the error message .......

**Comment by Nio9345** [comment]
Yes I got that .... And I didn't try it any further after I saw the lowest FW your trick can support is 9.6. I just want to know a way to access the Store without updating, because I wan't to redownload my games and the Smealum "eShop Access" isn't working for me.

**Comment by ghost** [comment]
Will there be an updated version to support v9 2 and possibly below? Like CTR httpwn v1.1 or something like that.

**Comment by ghost** [comment]
Sorry. Meant 9.2 typo

**Comment by yellows8** [comment]
https://github.com/yellows8/ctr-httpwn#supported-sysmodule-versions "It's unknown if/when auto-locating for the required sysmodule addresses would be implemented for supporting more versions." (Wanting to use this on v9.2 still doesn't make sense since there's really no reason to stay on that version besides "cfw"...)

**Comment by ghost** [comment]
Oh ok. Yea. I use 9.2 for CFW & Gateway. I just thought of an idea/request. It won't be the end of the world if it does not support 9.2-9.5 or below 9.2

**Comment by timX24968B** [comment]
I only really needed it on 9.2 since I still had some apps on my sysnand that I wanted to update but can't, and don't feel like moving them to emunand.
